code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -65,163 +86,339 @@
+											var/greed_type = pick(allTypes)
+											new greed_type(get_turf(sinPerson))
+									else
+										src.say("Your sin will be punished, [sinPerson]!")
+										sinPerson.reagents.add_reagent("nutriment",1000)
+								if("Pride")
+									src.say("Your sin, [sinPerson], is Pride.")
+									if(prob(50))
+										src.say("I will indulge your sin, [sinPerson].")
+										var/obj/item/weapon/twohanded/sin_pride/good = new/obj/item/weapon/twohanded/sin_pride(get_turf(sinPerson))
+										good.pride_direction = FALSE
+									else
+										src.say("Your sin will be punished, [sinPerson]!")
+										var/obj/item/weapon/twohanded/sin_pride/bad = new/obj/item/weapon/twohanded/sin_pride(get_turf(sinPerson))
+										bad.pride_direction = TRUE
+								if("Lust")
+									src.say("Your sin, [sinPerson], is Lust.")
+									if(prob(50))
+										src.say("I will indulge your sin, [sinPerson].")
+										var/obj/item/lovestone/good = new/obj/item/lovestone(get_turf(sinPerson))
+										good.lust_direction = FALSE
+									else
+										src.say("Your sin will be punished, [sinPerson]!")
+										var/obj/item/lovestone/bad = new/obj/item/lovestone(get_turf(sinPerson))
+										bad.lust_direction = TRUE
+								if("Envy")
+									src.say("Your sin, [sinPerson], is Envy.")
+									if(prob(50))
+										src.say("I will indulge your sin, [sinPerson].")
+										for(var/mob/living/carbon/human/H in player_list) // name lottery
+											if(H == sinPerson)
+												continue
+											if(prob(25))
+												spawn(10)
+													sinPerson.name = H.name
+													sinPerson.real_name = H.real_name
+													var/datum/dna/lottery = H.dna
+													lottery.transfer_identity(sinPerson, transfer_SE=1)
+													sinPerson.updateappearance(mutcolor_update=1)
+													sinPerson.domutcheck()
+									else
+										src.say("Your sin will be punished, [sinPerson]!")
+										var/sinPersonspecies = pick(species_list)
+										var/newtype = species_list[sinPersonspecies]
+										sinPerson.set_species(newtype)
+								if("Sloth")
+									src.say("Your sin, [sinPerson], is Sloth.")
+									if(prob(50))
+										src.say("I will indulge your sin, [sinPerson].")
+										sinPerson.drowsyness += 1000
+									else
+										src.say("Your sin will be punished, [sinPerson]!")
+										sinPerson.reagents.add_reagent("frostoil", 50)
+								if("Wrath")
+									src.say("Your sin, [sinPerson], is Wrath.")
+									if(prob(50))
+										src.say("I will indulge your sin, [sinPerson].")
+										sinPerson.change_mob_type(/mob/living/simple_animal/slaughter,get_turf(sinPerson),"Wrath Demon",1)
+									else
+										src.say("Your sin will be punished, [sinPerson]!")
+										sinPerson.reagents.add_reagent("lexorin", 100)
+										sinPerson.reagents.add_reagent("mindbreaker", 100)
 
 
 ///Sin related things
+
+/atom/movable/overlay/temp/hadesFlick

```

Comment:
`/obj/effect` not `/atom/movable`
Also density = 0 is redundant if using the full path of `/obj/effect/overlay/temp/hadesFlick`